src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +107,106 @@ const CrossPodWriteForm = () => {
-          />
-        </div>
-        <div style={formRowStyle}>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleCrossPodUpload} autoComplete="off">
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* Either way, also determine whether to update all other forms to that */}
+            <Typography htmlFor="cross-upload-doc">Upload document to username:</Typography>
+            <TextField
+              id="cross-upload-doc"
+              name="crossPodUpload"
+              label="Search Username"
+              required
+              value={username}
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder={selectedUser.username}
+            />
+          </FormControl>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expiration Date"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
+              />
+            </LocalizationProvider>
+          </FormControl>
           <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf, .gif"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+          <br />
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* <Typography htmlFor="upload-doc-desc">Enter description:</Typography> */}
```

Comment:
Can remove these two lines in 155 and 156.

Typography is not needed here. `<InputLabel>` could be the better choice, but that's already being handled by label and placeholder attributes in TextField so that's not needed either.